src/Transit.php

```diff
@@ -68,6 +68,24 @@ public function select(string $domainClass, array $whereEquals = []) : TransitSe
         );
     }
 
+    /**
+     * Attach a domain object to plan.
+     *
+     * @param \Atlas\Transit\object $domain
```

Comment:
This class doesn't exists `\Atlas\Transit\object`. Use `object` instead.

Reply:
Fixed ;)
Due to https://github.com/atlasphp/Atlas.Transit/pull/14